app/screens/SpecialVotingScreen.tsx

```diff
@@ -0,0 +1,163 @@
+import NotificationService from "../services/NotificationService";
+import AppStyles from "../styles/AppStyles";
+
+const SpecialVotingScreen = ({ session }: { session?: Session }) => {
+  const { t } = useTranslation();
+  const [isLoading, setIsLoading] = useState(true);
+  const [cfpResults, setCfpResults] = useState<CfpResult[]>();
+  const [canVote, setCanVote] = useState(false);
+  const [votes, setVotes] = useState<CfpVotes | undefined>();
+  const [isSaving, setIsSaving] = useState<{ number: Number; vote: CfpVote } | undefined>();
+
+  useAuthGuard(session);
+
+  const specialVotingCfp: CfpResult = {
+    number: 0,
+    title: t("specialVoting.title"),
+    type: VotingType.CFP,
+    dfiAmount: 0,
+    htmlUrl: "",
+    currentResult: ResultStatus.APPROVED,
+    totalVotes: {
+      total: 0,
+      possible: 0,
+      turnout: 0,
+      yes: 0,
+      neutral: 0,
+      no: 0
+    },
+    cakeVotes: {
+      total: 0,
+      yes: 0,
+      neutral: 0,
+      no: 0
+    },
+    voteDetails: {
+      yes: [],
+      no: [],
+      neutral: []
+    },
+    startDate: "2022-07-25T23:59:59.000Z",
+    endDate: "2022-08-02T23:59:59.000Z"
+  }
+
+  useEffect(() => {
+    Promise.all([getSettings(), getUserDetail()])
+      .then(([settings, user]) => {
+        const results: CfpResult[] = []
+
+        const startDate = new Date(specialVotingCfp.startDate)  
+        const currentDate = new Date()
+        const expiryDate = new Date(specialVotingCfp.endDate)
+
+        // specialVotingCfp automatically disappears after expiryDate
+        if (currentDate < expiryDate && currentDate > startDate) {
+          results.push(specialVotingCfp)
+        }
+
+        setCfpResults(results);
+        setVotes(votes);
+        setCanVote(user.stakingBalance >= 100);
+      })
+      .catch(() => NotificationService.error(t("feedback.load_failed")))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const onVote = (number: number, vote: CfpVote) => {
+    setVotes((votes) => {
+      votes = { ...(votes ?? {}), [number]: vote };
+
+      setIsSaving({ number, vote });
+      putCfpVotes(votes).finally(() => setIsSaving(undefined));
+
+      return votes;
+    });
+  };
+
+  return (
+    <AppLayout>
+      <View style={[AppStyles.container, AppStyles.alignCenter]}>
+        <H1 text={specialVotingCfp.title} style={AppStyles.center} />
+        <SpacerV height={50} />
+
+        {isLoading ? (
+          <Loading size="large" />
+        ) : (
+          <>
+            {cfpResults === undefined || (cfpResults.length === 0) && (
+              <>
+                <H3 text={t("specialVoting.ended")} />
+                <SpacerV height={50} />
+              </>
+            )}
+
+            {cfpResults
+              ?.sort((a, b) => a.number - b.number)
+              .map((result) => (
+                <View key={result.number} style={{ width: "100%" }}>
+                  <View style={AppStyles.halfWidth} >
```

Comment:
Half-width looks IMO a bit weird on smaller screens:
![image](https://user-images.githubusercontent.com/85513542/181038157-5941a105-4a25-4888-935d-28ef884875e5.png)
I would propose to go with 90%.

Reply:
There's a bug related to the halfWidth style. The background at the bottom is not blue:
![image](https://user-images.githubusercontent.com/85513542/181038454-6fa03707-aca6-415a-bd47-e5bef13b05d6.png)